src/@utils/aquarius.ts

```diff
@@ -170,6 +170,28 @@ export async function getAssetsFromDidList(
   }
 }
 
+export async function getAssetsFromDtList(
+  dtList: string[],
+  chainIds: number[],
+  cancelToken: CancelToken
+): Promise<any> {
```

Comment:
we should be able to be more precise here on the return type instead of `any`

Reply:
not sure we can, this is the result from elasticsearch. It's kinda complicated and not worth mapping now.  I created an issue at some point to see if we can handle some light mapping of es types but it gets complicated and not worth it.

Reply:
I mapped it to Asset[ ]. since we don't use accessDetails in this case I think it's enough